pkg/k8s/aggregator.go

```diff
@@ -73,8 +80,7 @@ func NewIngressAggregator(sources []Ingresswatcher) *IngressAggregator {
 		events: make(chan interface{}),
 	}
 	for _, s := range sources {
-		a.AddSource(s.Watcher)
-		a.IngressEndpoints = append(a.IngressEndpoints, s.IngressEndpoints...)
```

Comment:
`a.IngressEndpoints` is left out now - was never possible to have endpoints there, or what's the reason for dropping it?

Reply:
it was a growing list of all endpoints which randomly picked an item from ( which shouldn't be a growing list and should be unique per ingress resource )  
my mistake in the first place was to add it in `IngressAggregator` instead of `store` struct